### PR TITLE
Add basic Tasks index and creation

### PR DESCRIPTION
**Pull request by nogates**
output

### CODE REVIEW

File: frontend/app/templates/index.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{#link-to 'tasks.new' class="btn btn-primary btn-lg"}}
+  Create
```
Comment (atd) [inline frontend/app/templates/index.hbs:2]: What about i18n? Do you prefer adding it later?

File: Gemfile [hunk 2]
```diff
@@ -24,6 +24,7 @@ gem 'puma', '~> 3.0'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
+  gem 'pry'
```
Comment (atd) [inline Gemfile:27]: :+1:

File: config/initializers/serializers.rb [hunk 3]
```diff
@@ -1 +1,2 @@
+require 'active_model_serializers/register_jsonapi_renderer'
```
Comment (atd) [inline config/initializers/serializers.rb:1]: Really? Isn't it loaded by default?

**Comment by nogates** [comment]
@atd the test are broken because the data we are sending to the controller it's not in json api format. I was going to fix it, but, do you mind if we use `rspec`?

**Review by atd — COMMENTED** [review]
:dancer:

**Comment by atd** [comment]
I don't mind. We can switch to rspec if you are more comfortable with it

File: frontend/app/templates/index.hbs (hunk 1 above)
Comment (nogates) [reply frontend/app/templates/index.hbs:2]: Good point. I don't have much experience with it on ember. Will have a look and see if it's easy to implement, although I might do it in a different PR

File: config/initializers/serializers.rb (hunk 3 above)
Comment (nogates) [reply config/initializers/serializers.rb:1]: Nop!

**Comment by nogates** [comment]
cool! I need to do a few things now, but I might change the tests to use `rspec` in a couple of hours

**Comment by nogates** [comment]
also, if you want, go ahead and merge your PR... we are going to have a conflict with the `params.permit` logic in the `TasksController` but I am happy with resolving them

**Comment by atd** [comment]
Cool!

**Comment by atd** [comment]
You are a pro, I have never seen a video inside a PR before!

File: frontend/app/templates/index.hbs (hunk 1 above)
Comment (atd) [reply frontend/app/templates/index.hbs:2]: Cool!

**Comment by nogates** [comment]
> You are a pro, I have never seen a video inside a PR before!

it's a gif! :D

File: frontend/app/templates/index.hbs (hunk 1 above)
Comment (nogates) [reply frontend/app/templates/index.hbs:2]: https://github.com/jamesarosen/ember-i18n it looks like it's what we want

**Comment by nogates** [comment]
at last! :green_heart:

**Pull request merged**